### PR TITLE
data: kush6291.json

### PR DESCRIPTION
**Pull request by kush6291**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/kush6291.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+"name":"Kushagra Chavan (Test)",
+  "displayStatsPublic": true,
```
Comment (sital002) [inline data/kush6291.json:3]:
```suggestion
```
displayStatsPublic  is no longer required. Please remove it.

File: data/kush6291.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "type": "Personal",
+  "bio" "I am a DevOps Engineer",
+  "avatar": "https://github.com/kush6291.png"
+  "links": [
+      {
+        "name": "Follow me on twitter"
```
Comment (sital002) [inline data/kush6291.json:9]:
```suggestion
        "name": "Follow me on twitter",
```
Please add comma here

File: data/kush6291.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://twitter.com/Kush6291"
+      },
+    {
+        "name": "Follow me on github"
```
Comment (sital002) [inline data/kush6291.json:13]:
```suggestion
        "name": "Follow me on github",
```
please add comma here

File: data/kush6291.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/kush6291"
+      },
```
Comment (sital002) [inline data/kush6291.json:15]:
```suggestion
      }
```
The trailing comma is not required.

File: data/kush6291.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+"name":"Kushagra Chavan (Test)",
+  "displayStatsPublic": true,
+  "type": "Personal",
+  "bio" "I am a DevOps Engineer",
+  "avatar": "https://github.com/kush6291.png"
```
Comment (sital002) [inline data/kush6291.json:6]:
```suggestion
```
avatar property is no longer is required.

**Review by sital002 — CHANGES_REQUESTED** [review]

File: data/kush6291.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+"name":"Kushagra Chavan (Test)",
+  "bio" "I am a DevOps Engineer",
```
Comment (SaraJaoude) [inline data/kush6291.json:3]:
```suggestion
  "bio": "I am a DevOps Engineer",
```

**Review by SaraJaoude — APPROVED** [review]
Now the corrections to validate the json have been made this looks good and I can merge

**Pull request merged**